galsim/lsst/lsst_wcs.py

```diff
@@ -0,0 +1,1153 @@
+import numpy as np
+import warnings
+import galsim
+
+try:
+    import lsst.pex.logging as pexLog
+    import lsst.daf.base as dafBase
+    import lsst.afw.geom as afwGeom
+    import lsst.afw.cameraGeom as cameraGeom
+    import lsst.afw.image as afwImage
+    import lsst.afw.image.utils as afwImageUtils
+    import lsst.meas.astrom as measAstrom
+    from lsst.afw.cameraGeom import PUPIL, PIXELS, TAN_PIXELS, FOCAL_PLANE
+    from lsst.obs.lsstSim import LsstSimMapper
```

Comment:
One of these is importing matplotlib (which seems like an error, but ok).  On my machine this gives a warning:
```
>>> import galsim.lsst
/Users/Mike/lsst/DarwinX86/miniconda2/3.19.0.lsst4/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
```
Can you please wrap these in a context where warnings are ignored so the user doesn't need to see this.  Even better would be to fix whichever package in the stack is doing this, since there's no reason for that to be a top level import in any of these modules.